src/web3.ts

```diff
@@ -2,6 +2,14 @@
 import _ from 'lodash'
 import Web3 from 'web3'
 import {formatters} from 'web3-core-helpers'
+import * as T from 'io-ts'
+
+//https://eth.wiki/json-rpc/json-rpc-error-codes-improvement-proposal
```

Comment:
Is this implemented?

Reply:
Partially. Mantis has types definitions for these, so it can be used more widely. For now it only reports 100 in case of missing account data, which is enough for now and is a good groundwork for any further improvements if needed